Description: Correct pod NAME issue in LedgerSMB/DBObject/Vendor.pm
 Has a comma instead of a dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472577&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Vendor.pm
+++ b/LedgerSMB/DBObject/Vendor.pm
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-LedgerSMB::DBObject::Vendor, LedgerSMB Class for Vendors
+LedgerSMB::DBObject::Vendor - LedgerSMB Class for Vendors
 
 =head1 SYNOPSIS
 
